src/Controller/Blockexplorer/TransactionController.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Adshares\AdsOperator\Controller\Blockexplorer;
+
+use Adshares\AdsOperator\Controller\ApiController;
+use Adshares\AdsOperator\Repository\TransactionRepositoryInterface;
+use Adshares\AdsOperator\Document\Transaction;
+use Swagger\Annotations as SWG;
+use Nelmio\ApiDocBundle\Annotation\Operation;
+use Nelmio\ApiDocBundle\Annotation\Model;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class TransactionController extends ApiController
+{
+    /**
+     * TransactionController constructor.
+     * @param TransactionRepositoryInterface $repository
+     */
+    public function __construct(TransactionRepositoryInterface $repository)
+    {
+        $this->repository = $repository;
+    }
+
+    /**
+     * @Operation(
+     *     summary="List of transactions",
+     *     tags={"Blockexplorer"},
+     *
+     *     @SWG\Response(
+     *          response=200,
+     *          description="Returned when operation is successful",
+     *          @SWG\Schema(
+     *              type="array",
+     *              @SWG\Items(ref=@Model(type=Transaction::class))
+     *          )
+     *      ),
+     *     @SWG\Parameter(
+     *          name="sort",
+     *          in="query",
+     *          type="string",
+     *          description="The field used to order transactions"
+     *      ),
+     *      @SWG\Parameter(
+     *          name="order",
+     *          in="query",
+     *          type="string",
+     *          description="The field used to sort transactions"
+     *      ),
+     *      @SWG\Parameter(
+     *          name="limit",
+     *          in="query",
+     *          type="int",
+     *          description="The field used to limit number of transactions"
+     *      ),
+     *      @SWG\Parameter(
+     *          name="offset",
+     *          in="query",
+     *          type="int",
+     *          description="The field used to specify transactions offset"
+     *      )
+     * )
+     *
+     * @param Request $request
+     * @return Response
+     */
+    public function listAction(Request $request): Response
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method "listAction" to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1185)